sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -632,14 +645,38 @@ const StartHTLCEventListener = (
                     return;
                 }
             }
+            
             // All filters passed
             if (eventName === 'LockAsset' || eventName === 'LockFungibleAsset') {
+                //[!!!] New time code
+                const timeout = assetLockContractInfo.getLock().getExpirytimesecs();
                 const hashBase64 = assetLockContractInfo.getLock().getHashbase64();
+
+                //TODO: Get hash mechanism. Based on hash mechanism, make an instance of the class.
+                //Check SHA256 or SHA512
+                let hash: Hash;
+                
+                const hashMechanism = assetLockContractInfo.getLock().getHashMechanism(); 
+                if(hashMechanism === assetLocksPb.HashMechanism.SHA256) {
```

Comment:
Insert a space after `if` and before the opening paranthesis (`(`). Do that throughout your code. It'll make it consistent with existing code and be better looking. (Aesthetics are important in code too!)